src/main/java/core/basesyntax/service/FruitTransaction.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.service;
+
+public class FruitTransaction {
```

Comment:
The class `FruitTransaction` should have a constructor to initialize its fields instead of using setter methods. This is to ensure the immutability of the class instances which is a good practice.